fast_reliable.py

```diff
@@ -0,0 +1,62 @@
+import time
+from typing import Iterable
+
+
+def build_words_list_from_file_words(file_name: str) -> list:
```

Comment:
Please give that function a better name (for example, I think that the second words is unecessary). The typing hints and return type are great.